### PR TITLE
Add Project Loved: January 2022 Newspost

### PR DESCRIPTION
**Pull request by Pachiru**
Newspost for January Project Loved edition. (without forumgus post - need the australian seal of approval before pushing it)

### CODE REVIEW

File: news/2022-01-22-project-loved-january-2022.md
Comment (Walavouchey) [inline news/2022-01-22-project-loved-january-2022.md:296]:
```suggestion
The \[Envory's Hard\], \[Shikkou's Hard\] and \[Jakads' Extreme \[wip\]\] difficulties are *not* being nominated for Loved.\
```

**Comment by Walavouchey** [comment]
there are a few more formatting errors in the descriptions themselves that i've conveyed through dm (to fix on the forum post generation too), so i'll skim through this again later

**Comment by Walavouchey** [comment]
We may also want to come to a consensus on a few things:

In collection sets like [dressurf's 7k practice charts](https://osu.ppy.sh/beatmapsets/1232389#mania/2562052), the song titles can appear in the difficulty names (`Rock It`, `Decipher.#300` and `Meikaruza` are mentioned in the [relevant description](https://github.com/ppy/osu-wiki/blob/372f1169954e44dae46364bdd99c84bc33e178ec/news/2022-01-22-project-loved-january-2022.md?plain=1#L399-L405)). Difficulty names are not italicised, but song titles usually are. Which should take precedence?

Some have suggested using brackets (`[]`) to denote difficulty names in sentences (I've removed instances of this in this month's descriptions), which would help when they get long and have spaces, though that would also have to apply in the cases of [Normal] or [Hard] if we want to be consistent (and if not, the line to draw there is another discussion).

In beatmap links (and external links for that matter), we usually don't italicise anything (links already have a different colour, so additional emphasis serves no function). Not sure if we've done this elsewhere, but here we find the example of "Razzy's [*POETIC JUSTICE // AWAY* LN chart](https://osu.ppy.sh/beatmapsets/955714#mania/1996491)", which includes additional words in the link besides the title, where I've italicised the title to separate it. Another example is "[John Coltrane's *Giant Steps*](https://osu.ppy.sh/beatmapsets/1592995#mania/3253520)", where normally it'd be "[John Coltrane - Giant Steps](https://osu.ppy.sh/beatmapsets/1592995#mania/3253520)". If it wasn't a link, the whole thing would be in italics since the whole thing is referring to a piece of work.

Of course, we could reword every instance to avoid the question, but that would be an imposed limit on the writing department. We could also just always use italics, the ASC says that ["names of work or video games should be italicised"](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#italics) after all, but that would look stupid for e.g. mappool lists (which have the format of `[artist - title (mapper) [difficulty]](link)`), because a lot of italics would be excessive. As I said before, if it's a link, it's already emphasised by the different colour.

The main thing I'm asking about is the point about difficulty names. For the other instances, the way I've done it is consistent with what we've had before, so any changes to that would imply large changes elsewhere in the wiki, but I figured I'd bring it up anyway in case it sparks discussion (and warrants an issue). In any case, it might be worth clarifying some of this stuff in the ASC.

**Comment by Walavouchey** [comment]
I've moved my above points into an issue, and I'm ok with ignoring them for the purposes of this pr since they're only a concern if others actually do have differing opinions on the matter (so this is still just waiting on @Ephemeralis's approval for the forum posts)

**Review by Walavouchey — APPROVED** [review]

**Comment by Ephemeralis** [comment]
This is waiting on forum posts. Apparently there's problems with the script used to create them.

**Comment by Walavouchey** [comment]
waiting on forum post commit

**Review by Walavouchey — APPROVED** [review]

**Comment by Walavouchey** [comment]
Auto-merge can't merge without an approval (if that was your intention)

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**
